documentation/modules/auxiliary/scanner/couchdb/couchdb_login.md

```diff
@@ -2,6 +2,33 @@
 
 Apache CouchDB is a nosql database server which communicates over HTTP.  This module will enumerate the server and databases hosted on it.
 
+## CouchDB Docker container setup
+  1. `docker run -p 5984:5984 --env COUCHDB_USER=admin --env COUCHDB_PASSWORD=password apache/couchdb:latest`
```

Comment:
Instead of using the `latest` tag, we should probably inline the version that you tested with, i.e. `apache/couchdb:1.2.3` or whatever it was

Reply:
Good catch, never thought of that. I used latest in my command but will figure out what that value is and amend it 👍